modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -175,14 +177,16 @@ export default class NestedTabs extends Base {
 
 		$requestedContent[ settings.showTabFn ](
 			animationDuration,
-			() => {
-				elementorFrontend.elements.$window.trigger( 'elementor-pro/motion-fx/recalc' );
-				elementorFrontend.elements.$window.trigger( 'elementor/nested-tabs/activate', $requestedContent );
-			},
+			() => this.onShowTabContent( $requestedContent )
```

Comment:
@udidol Missing trailing comma

Reply:
There is no trailing comma in function calls, only in arrays